### PR TITLE
styles(TextField): avoid jumps while adding "focused" borders

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

- TextField: remove "jumping" when adding "focused" borders to TextField

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
* **#4587** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4587?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @guilhermespopolin and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/store/src/components/TextField/TextField.css.ts [hunk 1]
```diff
@@ -26,6 +26,7 @@ export const baseWrapper = style({
   display: 'flex',
   flexDirection: 'column',
   justifyContent: 'center',
+  border: '1px solid transparent',
```
Comment (alebedev) [inline apps/store/src/components/TextField/TextField.css.ts:29]: Can you comment why it's here?

**Review by alebedev — APPROVED** [review]

**Review by alebedev — APPROVED** [review]
Don't like this, but no idea how to do it better. `box-sizing: content-box` probably will just cause layout shift elsewhere

**Review by gustaveen — APPROVED** [review]
Great!

**Comment by guilhermespopolin** [comment]
> Don't like this, but no idea how to do it better. `box-sizing: content-box` probably will just cause layout shift elsewhere

Maybe an alternative solution would be to use `outline` for that. They don't cause any jumps. Gonna investigate.

**Comment by guilhermespopolin** [comment]
Have update it. It seems to work fine with outline,

**Pull request merged**